NGitLab/IMergeRequestClient.cs

```diff
@@ -45,6 +45,8 @@ public interface IMergeRequestClient
 
         IEnumerable<Author> GetParticipants(int mergeRequestIid);
 
+        IEnumerable<MergeRequestVersion> GetVersions(int mergeRequestIid);
```

Comment:
We try expose `async` methods when mapping a new method. In this case, only the return type need to be changed.

```suggestion
        GitLabCollectionResponse<MergeRequestVersion> GetVersionsAsync(int mergeRequestIid);
```

Reply:
👍